src/app/pages/receive/components/receive-item.tsx

```diff
@@ -0,0 +1,54 @@
+import { FiCopy } from 'react-icons/fi';
+
+import { Box, Button, ButtonProps, Flex, Stack } from '@stacks/ui';
+import { color, truncateMiddle } from '@stacks/ui-utils';
+
+import { Caption } from '@app/components//typography';
+import { Flag } from '@app/components/layout/flag';
+import { QrCodeIcon } from '@app/components/qr-code-icon';
+
+interface ReceiveItemProps extends ButtonProps {
+  address: string;
+  dataTestId?: string;
+  icon: React.JSX.Element;
+  onCopyAddress(): void;
+  onClickQrCode?(): void;
+  title: string;
+}
+export function ReceiveItem({
+  address,
+  dataTestId,
+  icon,
+  onCopyAddress,
+  onClickQrCode,
+  title,
+}: ReceiveItemProps) {
+  return (
+    <Flag img={icon} spacing="base">
+      <Flex justifyContent="space-between">
```

Comment:
We do have a `SpaceBetween` component if helpful here?

Reply:
Thanks! I just saw that now, useful 😄  

I think I'll leave it as is for now as I just moved this here and don't want to risk breaking something. I see that component has some other hardcoded props like `isInline` `alignItems='center'`